jobs/generation/PRTestGenerator.groovy

```diff
@@ -0,0 +1,482 @@
+// Import the utility functionality
```

Comment:
I think we need some visual scheme like the one currently added for nuget builds. It would be easier to review the workflow

Reply:
Yes, I'll prepare it.